ebu_tt_live/xsd1.1/ebutt_live.xsd

```diff
@@ -0,0 +1,403 @@
+	xmlns:ebuttp="urn:ebu:tt:parameters" targetNamespace="http://www.w3.org/ns/ttml"
+	xmlns:vc="http://www.w3.org/2007/XMLSchema-versioning" vc:minVersion="1.1">
+	<xs:import namespace="http://www.w3.org/XML/1998/namespace" schemaLocation="xml.xsd"/>
+	<xs:import namespace="http://www.w3.org/ns/ttml#styling" schemaLocation="styling.xsd"/>
+	<xs:import namespace="http://www.w3.org/ns/ttml#parameter" schemaLocation="parameter.xsd"/>
+	<xs:import namespace="urn:ebu:tt:metadata" schemaLocation="ebutt_metadata.xsd"/>
+	<xs:import namespace="urn:ebu:tt:style" schemaLocation="ebutt_styling.xsd"/>
+	<xs:import namespace="urn:ebu:tt:datatypes" schemaLocation="ebutt_datatypes.xsd"/>
+	<xs:import namespace="urn:ebu:tt:parameters" schemaLocation="ebutt_parameters.xsd"/>
+	<xs:import namespace="http://www.w3.org/ns/ttml#metadata" schemaLocation="metadata.xsd"/>
+	<xs:complexType name="tt">
+		<xs:annotation>
+			<xs:documentation>Root element of an EBU-TT XML document.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="head" type="tt:head"/>
+			<xs:element name="body" type="tt:body" minOccurs="0"/>
+		</xs:sequence>
+		<xs:attribute ref="xml:space">
+			<xs:annotation>
+				<xs:documentation>Indicates the author’s intention of white space handling within
+					the content of the EBU-TT document.</xs:documentation>
+			</xs:annotation>
+		</xs:attribute>
+		<xs:attribute ref="ttp:timeBase" use="required"/>
+		<xs:attribute ref="ttp:frameRate" use="optional"/>
+		<xs:attribute ref="ttp:frameRateMultiplier" use="optional"/>
+		<xs:attribute ref="ttp:markerMode" use="optional"/>
+		<xs:attribute ref="ttp:dropMode"/>
+		<xs:attribute ref="ttp:clockMode"/>
+		<xs:attribute ref="ttp:cellResolution"/>
+		<xs:attribute ref="ebuttp:sequenceIdentifier" use="required"/>
+		<xs:attribute ref="ebuttp:sequenceNumber" use="required"/>
+		<xs:attribute ref="ebuttm:authoringDelay" use="optional"/>
+		<xs:attribute ref="ebuttp:authorsGroupIdentifier" use="optional"/>
+		<xs:attribute ref="ebuttp:authorsGroupControlToken" use="optional"/>
+		<xs:attribute ref="ebuttm:authorsGroupControlRequest" use="optional"/>
+		<xs:attribute ref="ebuttp:referenceClockIdentifier" use="optional"/>
+		<xs:attribute ref="tts:extent">
+			<xs:annotation>
+				<xs:documentation>Defines the width and height of the active video the subtitles
+					were authored for. Only length expressions in pixels shall be
+					used.</xs:documentation>
+			</xs:annotation>
+		</xs:attribute>
+		<xs:attribute ref="xml:lang" use="required">
+			<xs:annotation>
+				<xs:documentation>The language for which the EBU-TT document is prepared. (STL
+					mapping: Language Code (LC))</xs:documentation>
+			</xs:annotation>
+		</xs:attribute>
+		<xs:assert test="@ttp:timeBase = 'smpte' and exists(@ebuttp:referenceClockIdentifier)
+			or (@ttp:timeBase = 'clock' and @ttp:clockMode = 'local' and exists(@ebuttp:referenceClockIdentifier))
```

Comment:
Did you test this with no ttp:clockMode attribute present? The default interpretation if it is absent is the value `"utc"` according to https://www.w3.org/TR/ttml1/#parameter-attribute-clockMode so the desired behaviour is that second bracketed clause returns false if the `ttp:clockMode` attribute is absent.